### PR TITLE
Release 0.32.1 and changelog fixes from prev release

### PR DESCRIPTION
**Pull request by ishustava**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,60 +1,12 @@
 ## Unreleased
 
+## 0.32.1 (June 29, 2021)
+
 BUG FIXES:
 * ACLs: Support connecting to external Consul servers with an SNI header. [[GH-1005](https://github.com/hashicorp/consul-helm/pull/1005)]
 
 ## 0.32.0 (June 22, 2021)
 
-FEATURES:
```
Comment (ishustava) [inline CHANGELOG.md:8]: Reordering and putting breaking changes first.

Comment (kschoche) [reply CHANGELOG.md:8]: awesome, good call on this!

**Review by kschoche — APPROVED** [review]
🔥

**Review by thisisnotashwin — APPROVED** [review]
Thanks for fixing the order on the Changelog 🙏

**Pull request merged**